src/main/java/com/yurjinia/project_structure/column/dto/ColumnDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.yurjinia.project_structure.column.dto;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ColumnDTO {
+
+    @NotNull
+    private String name;
+
+    private long columnPosition;
```

Comment:
do we need to work with `columnPosition` in "create column" endpoint?

Reply:
We are not workin with `columnPosition` in create column endpoint.
Should i create new DTO if it is not used in create endpoint?

Reply:
yes, then we need to create CreateColumnRequest class